examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/paywall/PaywallScreenViewModel.kt

```diff
@@ -88,10 +89,12 @@ class PaywallScreenViewModelImpl(
     override fun onPurchaseError(error: PurchasesError) {
         val value = _state.value
         if (value is PaywallScreenState.Loaded) {
-            _state.update {
-                value.copy(
-                    dialogText = "There was an error purchasing:\n${error.message}",
-                )
+            if (error.code != PurchasesErrorCode.PurchaseCancelledError) {
```

Comment:
It would be nice to add unit tests to these view models.

Reply:
Well, I definitely agree for the RevenueCatUI module. For paywall-tester... it might be good but I think it's less important, since it's just meant to be used internally.

Reply:
Ooooh I thought this was on `RevenueCatUI` 😅